robot/Cumulus/resources/NPSPSettingsPageObject.py

```diff
@@ -247,4 +247,25 @@ def verify_status_to_state_mappings(self, **kwargs):
                 print(f"actual mapping found for {key} is {actual_value}")
                 assert value == actual_value, "Expected {} value to be {} but found {}".format(key,value, actual_value)
             else:
-                print("Right keys under status mapping fields table not found")    
\ No newline at end of file
+                print("Right keys under status mapping fields table not found")
+
+    @capture_screenshot_on_error
+    def set_batch_number_format_status(self,number_format,status):
+        """Deactivates the given batch number format by clicking on dropdown and selecting option
+        And verifies that the format has been deactivated"""
+        locator=npsp_lex_locators['npsp_settings']['table_dropdown'].format(number_format)
+        self.selenium.get_webelement(locator).click()
+        if status.lower()=='deactivate':
+            self.selenium.wait_until_page_contains("Deactivate")
+            self.selenium.click_link("Deactivate")
+            self.builtin.sleep(1,"To allow time for format to be deactivated")
+            self.npsp.verify_table_contains_row("datatable",number_format,Active="False")
+        elif status.lower()=='activate':
```

Comment:
Is the syntax 'elif', or should it be 'else if' ?

Reply:
in python we use 'elif' for 'else if'